### PR TITLE
Task-54468: User exo-user component for kudos activity and comment

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityLikerItem.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
       :identity="liker"
       avatar-class="me-2"
       size="42"
-      class="pl-3 pt-2 pb-1"
+      :extra-class="'pl-3 pt-2 pb-1'"
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityLikerItem.vue]:
```suggestion
      extra-class="pl-3 pt-2 pb-1"
```

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue [hunk 2]
```diff
@@ -93,12 +93,24 @@
...
           :class="useEllipsisOnTitle && 'text-truncate-2' || ''"
           class="font-weight-bold text-color mx-0 mt-0 mb-2 text-wrap text-break">
         </div>
+        <div 
+          v-else
+          class="d-flex align-center text-color mx-0 mt-0 mb-2 text-wrap text-break">
+          <span class="font-weight-bold" v-sanitized-html="title"></span>
+          <exo-user
+            :profile-id="identityReceiver"
+            :extra-class="'ms-2'"
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue]:
```suggestion
            extra-class="ms-2"
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityLikerItem.vue (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityLikerItem.vue]: Done!

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue (hunk 2 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue]: Done!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**